app/assets/javascripts/admin/addon/components/schema-theme-setting/editor.gjs

```diff
@@ -230,58 +231,74 @@ export default class SchemaThemeSettingNewEditor extends Component {
           this.args.themeId
         );
       })
-      .catch(popupAjaxError)
+      .catch((e) => {
+        if (e.jqXHR.responseJSON && e.jqXHR.responseJSON.errors) {
+          this.validationErrorMessage = e.jqXHR.responseJSON.errors[0];
```

Comment:
`this.validationErrorMessage` gets cleared somewhere, right?

e.g. if the next `saveChanges` happens and it hits the `else` case.

Reply:
For now when saving is successful, we redirect back to the themes#show route. If it isn't we want the error message to stay displayed until stuff is saved the next time.